lua/lspconfig/volarvuels.lua

```diff
@@ -0,0 +1,88 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'volarvuels'
+
+-- TODO: This is the name if you download volar-server from the AUR
+-- need to wait for https://github.com/johnsoncodehk/volar/issues/458
+local bin_name = 'volar-server'
+
+-- TODO: Figure out how to load this dynamically
+local typescript_server_path = '/home/artur/.npm-global/lib/node_modules/typescript/lib/tsserverlibrary.js'
```

Comment:
Path to tsserver is hardcoded. I need to figure out how to use the $NPM_PREFIX to look for a global installation of tssesrver OR look in the current workspace.

Another problem is that supposedly Volar doesn't work very well with TypeScript 4.4?

Reply:
I could put some .lua script that'd scan the filesystem in one of the on_setup hooks nvim-lspconfig has. I'm just pretty sure I won't be able to call shell commands like `npm get prefix` or `npx` to know where to look for node_modules (global and local respectively)